src/cf-vars.less

```diff
@@ -44,15 +47,18 @@
     - cf-core
 */
 
-// Sizing variables 
+// Sizing variables
 
 @base-font-size-px:             16px;
 @base-line-height-px:           22px;
 @base-line-height:              unit(@base-line-height-px / @base-font-size-px);
 @mobile-max:                    599px;
 @tablet-min:                    600px;
+@tablet-max:                    800px;
+@desktop-min:                   801px;
+@wall-min:                      1200px;
```

Comment:
That's my problem with naming it `tv`, you're mentally tied to the device size and not the breakpoint at which the layout starts falling apart. In our case the breakpoint just happens to be 1200px, but that doesn't mean we need to pill box there. We could (and probably should) design a true layout for larger than 1200px screens.